### PR TITLE
Rate test cases

### PR DESCRIPTION
**Pull request by Kok-je**
Added test cases for `Rate` class.

### CODE REVIEW

**Comment by kevinchangjk** [comment]
Are you adding the test cases for the `rate` command and parser as well?

File: docs/UserGuide.md [hunk 1]
```diff
@@ -1,751 +1 @@
...
| **Rate**            | `rate id/ID r/RATING`<br> e.g. `rate id/1 r/3`                                                                                                                                       |
| **View Department** | `view-department DEPARTMENT`<br> e.g. `view-department Finance`                                                                                                                      |
| **Help**            | `help`                                                                                                                                                                               |
| **Exit**            | `exit`                                                                                                                                                                               |
| **Clear**           | `clear`                                                                                                                                                                              |
```
Comment (kevinchangjk) [inline docs/UserGuide.md]: Sorry but what is this.

**Review by kevinchangjk — COMMENTED** [review]
A bit of a change in the UG?

File: docs/UserGuide.md (hunk 1 above)
Comment (Kok-je) [reply docs/UserGuide.md]: how windows interpret a new line, the clrf thing

Comment (Kok-je) [reply docs/UserGuide.md]: rate command different branch different commit

**Review by kevinchangjk — APPROVED** [review]
LGTM.

**Pull request merged**